### ISSUE TITLE
MesloLGS NF font not showing properly

### ISSUE THREAD

**Issue by GoodPal** [bajrangCoder/acode-plugin-acodex#78]
**Describe the bug**
MesloLGS NF font not showing properly in Acode

![IMG_20241106_214916_134](https://github.com/user-attachments/assets/0837fbdf-4b7c-4512-9f23-2b6799756ba2)

**Comment by bajrangCoder** [comment]
Provide more context , as it's working fine for me.

Check whether fonts are downloaded or not , check AcodeX font family settings

**Comment by bajrangCoder** [comment]
AcodeX download these fonts in different directory, it doesn't uses termux one. Check whether it is downloaded or not at (`internal storage/Android/data/com.foxdebug.acode/files/acodex_fonts`)

And also is other fonts working?

**Comment by GoodPal** [comment]
Yes, all fonts work fine and make changes, but I could not access this path and see inside this folder, I was not allowed to enter this folder.
